libcxx/test/support/test_range.h

```diff
@@ -94,4 +96,13 @@ concept CanBePiped = requires(View&& view, T&& t) {
   { std::forward<View>(view) | std::forward<T>(t) };
 };
 
+template <class T, class U>
```

Comment:
Can you please readd the comment that shows this is a standard (exposition-only) concept?